resources/views/pages/ticket/[ticket].blade.php

```diff
@@ -0,0 +1,423 @@
+            @if ($permissions >= Permissions::Developer)
+                @if (TicketState::isOpen($ticket->ReportState))
+                    <select name="action" required>
+                        <option value="" disabled selected hidden>Choose an action...</option>
+                        <option value="{{ TicketAction::Resolved }}">Resolve as fixed (add comments about your fix above)</option>
+                        @if ($ticket->ReportState === TicketState::Open)
+                            @php
+                                $lastComment = null;
+                                foreach ($commentData as $comment) {
+                                    if ($comment['User'] != 'Server') {
+                                        $lastComment = $comment;
+                                    }
+                                }
+                            @endphp
+                            @if ($lastComment != null && ($lastComment['User'] === $user->User || $lastComment['User'] === $ticket->achievement->Author))
+                                <option value="{{ TicketAction::Request }}">Transfer to reporter - {{ $ticket->reporter->User }}</option>
+                            @endif
+                        @else
+                            <option value="{{ TicketAction::Reopen }}">Transfer to author - {{ $ticket->achievement->Author }}</option>
+                        @endif
+                        <option value="{{ TicketAction::Demoted }}">Demote achievement to Unofficial</option>
+                        <option value="{{ TicketAction::Network }}">Close - Network problems</option>
+                        <option value="{{ TicketAction::NotEnoughInfo }}">Close - Not enough information</option>
+                        <option value="{{ TicketAction::WrongRom }}">Close - Wrong ROM</option>
+                        <option value="{{ TicketAction::UnableToReproduce }}">Close - Unable to reproduce</option>
+                        <option value="{{ TicketAction::ClosedMistaken }}">Close - Mistaken report</option>
+                        <option value="{{ TicketAction::ClosedOther }}">Close - Another reason (add comments above)</option>
+                    </select>
+                    <button class='btn'>Perform action</button>           
+                @else
+                    <input type="hidden" name="action" value="{{ TicketAction::Reopen }}">
+                    <button class='btn'>Reopen this ticket</button>
+                @endif
+            @elseif ($user?->id === $ticket->reporter->id)
+                @if (TicketState::isOpen($ticket->ReportState))
+                    <input type="hidden" name="action" value="{{ TicketAction::ClosedMistaken }}">
+                    <button class='btn'>Close as mistaken report</button>
+                @endif
+            @endif
+        </form>
+
+        @if ($permissions >= Permissions::JuniorDeveloper)
```

Comment:
Ideally this would be driven by something in our achievement policy, but #2328 is likely a hold-up here.